src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -281,6 +289,13 @@ public MqttServerFactory(
         this.clientId = config.clientId();
         this.publishTimeoutMillis = SECONDS.toMillis(config.publishTimeout());
         this.connectTimeoutMillis = SECONDS.toMillis(config.connectTimeout());
+        this.sessionExpiryInterval = config.sessionExpiryInterval();
+        this.maximumQos = config.maximumQos();
+        this.retainAvailable = config.retainAvailable() ? (byte) 1 : 0;
+        this.wildcardSubscriptionAvailable = config.wildcardSubscriptionAvailable() ? (byte) 1 : 0;
+        this.subscriptionIdentifierAvailable = config.subscriptionIdentifierAvailable() ? (byte) 1 : 0;
+        this.sharedSubscriptionAvailable = config.sharedSubscriptionAvailable() ? (byte) 1 : 0;
```

Comment:
Simplify `retainAvailable`, `wildcardSubscriptionAvailable`, `subscriptionIdentifierAvailable` and `sharedSubscriptionAvailable` to booleans called `retainedMessages`, `wildcardSubscriptions`, `subscriptionIdentifiers` and `sharedSubscriptions`?